src/test/java/pl/szczesniak/dominik/whattowatch/movies/domain/TestMoviesToWatchServiceConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package pl.szczesniak.dominik.whattowatch.movies.domain;
+
+public class TestMoviesToWatchServiceConfiguration {
+
+    static MoviesToWatchService moviesToWatchService(MoviesRepository moviesRepository, UserProvider userProvider) {
```

Comment:
finali brakuje

Reply:
i nie chciałbym żebyś w testowej konfiguracji przekazywał MoviesRepo... testowa konfiguracja powinna sama tworzyć InMemoryRepo przez co chowamy referencję do repository przez co nikt nie będzie mógł w teście weryfikować co stało się w repozytorium po wykonaniu operacji, które testujemy (wymusza nam to lepsze testy BDD skupiające się na zachowaniach, a nie na tym jakie dane są w bazie danych/repo)